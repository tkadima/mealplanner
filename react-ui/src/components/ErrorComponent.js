import { Message } from 'semantic-ui-react'

const ErrorComponent = ({message}) => {
    return (
        <Message error>
            <Message.Header>Error: {message}</Message.Header>
        </Message>
    )
}
export default ErrorComponent