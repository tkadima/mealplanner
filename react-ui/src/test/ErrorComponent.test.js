import React from 'react'
import { shallow } from 'enzyme'
import ErrorComponent from '../components/ErrorComponent'

test('should test ErrorComponent component', () => {
    const wrapper = shallow(<ErrorComponent />);
    expect(wrapper).toMatchSnapshot();
   });