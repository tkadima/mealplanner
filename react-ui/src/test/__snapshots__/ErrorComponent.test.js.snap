// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test ErrorComponent component 1`] = `ShallowWrapper {}`;
